docs/linux.md

```diff
@@ -118,25 +121,25 @@ repository](https://github.com/flathub/org.gaphor.Gaphor).
 
 1. [Install Flatpak](https://flatpak.org/setup)
 
-1. Install flatpak-builder
+2. Install flatpak-builder
```

Comment:
It is fine that you renumbered these, but Markdown does automatic numbering. This allows you to insert another number without manually renumbering everything below it.

Reply:
Yes, I realized later but it was more effort changing it back ;)